app.py

```diff
@@ -203,8 +203,8 @@ def show_all_services():
 
 @app.route('/api/service/<int:service_id>')
 def get_service_data(service_id):
-    return ServiceService.get_service(service_id)    
-
+    return ServiceService.get_service(service_id)  
+    
 @app.route('/invoices/add', methods=["POST"])
```

Comment:
no need to have the verb as well since this is a post route
```suggestion
@app.route('/invoices', methods=["POST"])
```